plugins/remote-config/frontend/public/javascripts/countly.views.js

```diff
@@ -2,13 +2,338 @@
 
 (function() {
     var FEATURE_NAME = "remote_config";
+    var _mixins = countlyVue.mixins;
 
+    var COLOR_TAG = [{
+        value: 1,
+        label: "#6C47FF"
+    },
+    {
+        value: 2,
+        label: "#39C0C8"
+    },
+    {
+        value: 3,
+        label: "#F96300"
+    },
+    {
+        value: 4,
+        label: "#F34971"
+    },
+    {
+        value: 5,
+        label: "#F5C900"
+    }
+
+    ];
+
+    var DATA_TYPES = {
+        d: CV.i18n("remote-config.type.d"),
+        n: CV.i18n("remote-config.type.n"),
+        bl: CV.i18n("remote-config.type.bl"),
+        s: CV.i18n("remote-config.type.s"),
+        l: CV.i18n("remote-config.type.l")
+    };
+
+    var ConditionStats = countlyVue.views.BaseView.extend({
+        template: '	<table class="cly-vue-remote-config-percentages-breakdown">\
+			<thead>\
+				<tr>\
+					<th class="cly-vue-remote-config-percentages-breakdown__sequence__heading bu-pl-2">\
+						#\
+					</th>\
+					<th class="cly-vue-remote-config-percentages-breakdown__heading">\
+						{{i18n("remote-config.condition")}}\
+					</th>\
+                    		<th class="cly-vue-remote-config-percentages-breakdown__heading">\
+						{{i18n("remote-config.percentage")}}\
+					</th>\
+				</tr>\
+			</thead>\
+			<tbody>\
+                <tr><td><div class="cly-vue-remote-config-percentages-breakdown__sequence bu-py-1 bu-px-2">1</div></td><td><div class="bu-is-align-items-center	bu-is-flex cly-vue-remote-config-percentages-breakdown__data bu-p-1 cly-vue-remote-config-percentages-breakdown__default-value"><span class="bu-ml-2 bu-mr-3">{{i18n("remote-config.default-value")}}</span><span class="cly-vue-remote-config-percentages-breakdown__default-value__value bu-py-1 bu-px-2">{{defaultValue.value}}</span></div></td><td>{{defaultValue.percentage}} {{i18n("remote-config.percent.of.total")}}</td></tr>\
+				<tr v-if="isDrillEnabled" v-for="(condition, i) in conditions" :key="i">\
```

Comment:
does conditions array contain the default value aswell?

Reply:
if yes, aren't we supposed to show default value when drill is disabled? will default value be shown?

Reply:
yes it will be shown :)